rackcity/views/it_model_views.py

```diff
@@ -496,7 +496,7 @@ def model_fields(request):
 @api_view(['GET'])
 def i_am_admin(request):
     print("yeah")
-    if(request.user.is_superuser):
+    if(request.user.is_staff):
```

Comment:
is staff the new backend representation of admin? Do we also need to be able to distinguish between the "admin admin" and other admin users?

Reply:
see above comment!! staff & superuser are Django definitions